datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -867,11 +867,11 @@ void shutdownConnectionStatusReceiver() {
     }
 
     /**
-     * see {@link WiFiSurveyor#isConnected()}
+     * see {@link WiFiSurveyor#isConnectedToSyncableNetwork()}
      */
     @SuppressWarnings("unused") // Allows implementing apps (CY) to only trigger sync manually when connected
     public boolean isConnected() {
-        return getWiFiSurveyor().isConnected();
+        return surveyor.isConnectedToSyncableNetwork();
```

Comment:
Is semantically clearer as there is a "isConnected" method in Network which returns true when the network is connected, but we want this method to only return true when there is a syncable network connected (i.e. wifi only? then only when such a network is connected)